runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/windmill/work/budget/EvenGetWorkBudgetDistributorTest.java

```diff
@@ -192,17 +82,17 @@ public void testDistributeBudget_distributesBudgetEvenlyIfPossible() {
     streams.forEach(
```

Comment:
just skip the math in the test and inline the right values?
The math is just copying what we have in the impl, if there is some bug in the impl hard coding the values at least is a sanity check.

ditto below.

Reply:
done